ios/ios_testing_references.md

```diff
@@ -18,47 +18,55 @@ UI testing is the process of ensuring proper functionality of the UI for a given
 
 The iOS Team's default approach is to use the XCTest Framework when writing tests. Given that XCTest is an official Apple supported testing framework our guidance is to use XCTest.
 
-The iOS Team has experience with the following additional testing frameworks, which can be used however the guidance is to use XCTest.
-
-* Quick/Nimble (https://github.com/Quick/Nimble) 
+The iOS Team has experience with additional testing frameworks, such as Quick/Nimble. However, while these can be used, the guidance is to use XCTest until you have fully grasped the process of writing tests.
 
 ## What to Test
 
-The iOS team is currently focused on writing unit tests against Models and ViewModels. By adopting an MVVM architecture, most business logic will be extracted from the ViewController and included in a ViewModel, thereby making this business logic more testable. For a more detailed look at Testing and MVVM see this related [reference] (https://github.com/IntrepidPursuits/sherpa/blob/master/ios/MVVM_Testing.md). 
+The iOS team is currently focused on writing unit tests against Models and ViewModels. By adopting an MVVM architecture, most business logic will be extracted from the ViewController and included in a ViewModel, thereby making this business logic more testable. For a more detailed look at Testing and MVVM see this related [reference] (https://github.com/IntrepidPursuits/sherpa/blob/master/ios/MVVM_Testing.md).
 
 ## Testing Tools
 
-Provided below is a list of commonly used tools when writing tests. 
+Provided below is a list of commonly used tools when writing tests.
+
+### OHHTTPStubs - Web stubbing
+
+Stubbing webcalls is required when unit testing portions of a codebase that relied on web resources. The desired stubbing tool is [OHHTTPStubs](https://github.com/AliSoftware/OHHTTPStubs). OHHTTPStubs allow for stubbing of specific webcalls or all web traffic. Stubbed webcalls are intercepted and allow for a static file (JSON/XML/etc) to be returned.
```

Comment:
relied -> rely
I know there is a knowledge gap here having watching some people try to write tests against web endpoints and just calling the real thing and not even the slightest idea that might be a good idea. I think we need to provide explanation as to why.

Reply:
Also should expand they will need to interact with the backend team to get fixture that represent the results of calls.

Reply:
👍